roles/openshift_node/tasks/config.yml

```
@@ -70,6 +70,40 @@
     state: directory
   register: temp_dir
 
+- name: Copy 00-runc-crc.yaml ContainerRuntimeConfig
+  copy:
+    content: >
+      apiVersion: machineconfiguration.openshift.io/v1
+      kind: ContainerRuntimeConfig
+      metadata:
+        name: openshift-ansible-use-runc
+      spec:
+        machineConfigPoolSelector:
+          matchLabels:
+            pools.operator.machineconfiguration.openshift.io/{{ openshift_node_machineconfigpool }}: ''
+        containerRuntimeConfig:
+          defaultRuntime: "runc"
+    dest: "{{ temp_dir.path }}/openshift-ansible-runc-crc.yaml"
+
+- name: Apply use-runc ContainerRuntimeConfig
+  command: >
+    oc apply -f "{{ temp_dir.path }}/openshift-ansible-runc-crc.yaml"
+  delegate_to: localhost
+  register: oc_apply
+  until:
+  - oc_apply.rc = 0
+  - oc_apply.stdout ~= "(created|configured|unchanged)"
+  retries: 36
+  delay: 5
+
+# it'd probably be better to retrieve the content includes "use-runc" from above?
+# no idea what a reasonable time for MachineConfig to regenerate looks like
+- name: Wait 30s if oc_apply
+  pause: 
```

```suggestion
  pause:
```
yaml lint is complaining about the trailing space here, after `pause:`
can fix on the next round, after these tests...